swiss/src/app/round/round.component.ts

```diff
@@ -329,6 +329,60 @@ export class RoundComponent implements OnInit {
     )
   }
 
+  setQuestions() {
+    this.dialog.open(DeleteConfirmDialogComponent, 
+      {data: "všech dekretů v tomto kole a vytvoření nových prázdných (1 dekret na jedno volební právo v Landsraadu)"})
+      .afterClosed()
+      .subscribe(
+        result => {
+          if (result) {
+            // delete old questions and answers
+            this.db.list("landsraad/questions", ref => ref.orderByChild("roundId").equalTo(this.roundId)).snapshotChanges().pipe(
+              take(1),
+              map(
+                snapshots => {
+                  snapshots.forEach(
+                    snapshot => {
+                      this.db.object(`landsraad/questions/${snapshot.key}`).remove()
+                      this.db.object(`landsraad/answers/${snapshot.key}`).remove()
+                    }
+                  )
+                }
+              )
+            ).subscribe()
+            // create new questions and answers for each votingRight
+            this.db.list("landsraad/votingRights").valueChanges().pipe(
+              take(1),
+              map(
+                votingRights => {
+                  votingRights.forEach(
+                    votingRight => {
+                      let delegateId = votingRight["controlledBy"]
+                      let ref = this.db.list("landsraad/questions/").push({
+                        questionType: "",
+                        name: "",
+                        byDelegateId: delegateId,
+                        byVotingRight: votingRight["name"],
```

Comment:
This is a bit of code smell - saving name into db and not id. But if it means larger refactor, let's keep it.

Reply:
I will see how big refactor it is. I think at this point not so big, so I could still do it.

Reply:
Done.